Task1.1/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿public class Program
+{
+    public static void Main(string[] args)
+    {
+        Console.Write("Input first number: ");
+        int a = int.Parse(Console.ReadLine());
+
+        Console.Write("Input second number: ");
+        int b = int.Parse(Console.ReadLine());
+
+        for (int i = a; i <= b; i++)
+        {
+            var duodecimal = DecimalToDuodecimal(i);
+            if (HasTwoASymbols(duodecimal))
+            {
+                Console.WriteLine(i);
+            }
+        }
+    }
+    public static bool HasTwoASymbols(string num)
+    {
+        int count = 0;
+
+        foreach (var x in num)
+        {
+            if (x == 'A')
+            {
+                count++;
+            }
+        }
+        return count == 2;
+    }
+
```

Comment:
If you create a method to convert from one numerical system to other make it more general -> from decimal to any other system. But in this task is not necessary to make a conversion, it will be enough just to calculates number of reminders equal to 10

Reply:
fixed